src/ccl_correlation.c

```diff
@@ -382,7 +402,7 @@ static void ccl_tracer_corr_legendre(ccl_cosmology *cosmo,
       wtheta[i]=0;
```

Comment:
there's no parallelization here. Is this intentional? As above, I doubt we'll use this in practice, and even if we do, it's probably not super-high priority that we parallelize this now. Just making sure this is not an oversight.

Reply:
I've been trying to add openmp only where it seems to help. I don't think I chcked this section of code, so this was def an oversight. I will take a look.

Reply:
ok, great. As I said, zero priority, I think.

Reply:
This is done.